### PR TITLE
🚩 Add Feature Flag to Bypass Auth in Local Dev (#370)

### PR DESCRIPTION
**Pull request by mistryrn**
### Consent UI
* Adds `AUTH_DISABLED` env var which bypasses auth on protected routes for local dev
* Update README "Environment Variables" table

### Special Instructions
In order to bypass auth during local dev, add the following to your `.env.local` in `consent-ui`:
```.env
AUTH_DISABLED=true
```

## PR Readiness Checklist

- [ ] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,6 @@
 
 [![TypeScript](https://img.shields.io/badge/types-%20TypeScript-blue)](https://www.typescriptlang.org/)
 
-<!-- | Release    | Build Status                                                                                                                                                                    |
```
Comment (mistryrn) [inline README.md:5]: Removing this block from the README since its been unused since we started the project 😛

File: README.md [hunk 2]
```diff
@@ -75,39 +70,40 @@ To run the setup locally, ensure you have provided the **required** environment
...
-| `phi-das`     | `DATABASE_URL`         | URL for the PHI DAS               | `string` | Required | `postgres://postgres:postgres@localhost:5432/phi_db`                                                                                                           |
-| `phi-das`     | `PORT`                 | Port number for the PHI DAS       | `number` | Optional | `8083`                                                                                                                                                         |
-| `pi-das`      | `DATABASE_URL`         | URL for the PI DAS                | `string` | Required | `postgres://postgres:postgres@localhost:5432/pi_db`                                                                                                            |
-| `pi-das`      | `PORT`                 | Port number for the PI DAS        | `number` | Optional | `8082`                                                                                                                                                         |
```
Comment (mistryrn) [inline README.md:110]: Mostly just whitespace/formatting updates here, only one new value added to the table (highlighted in next comment)

File: README.md [hunk 3]
```diff
@@ -75,39 +70,40 @@ To run the setup locally, ensure you have provided the **required** environment
...
+| `consent-das` | `PORT`                 | Port number for the Consent DAS                                                                                                                                                                                                                         | `number`  | Optional | `8085`                                                                                                                                                         |
+| `consent-ui`  | `AUTH_DISABLED`        | Disables auth for local UI development.                                                                                                                                                                                                                 | `boolean` | Optional |                                                                                                                                                                |
```
Comment (mistryrn) [inline README.md:83]: This is the only new value added to the table, the rest is just whitespace/formatting

File: README.md (hunk 1 above)
Comment (anncatton) [reply README.md:5]: wow i don't even remember this being there!

File: apps/consent-ui/src/config/appConfig.ts [hunk 4]
```diff
@@ -30,6 +31,7 @@ export type AppConfig = {
 };
 
 export const defaultAppConfig: AppConfig = {
+	AUTH_DISABLED: undefined,
```
Comment (anncatton) [inline apps/consent-ui/src/config/appConfig.ts:34]: should we just set this to `false`?

Comment (mistryrn) [reply apps/consent-ui/src/config/appConfig.ts:34]: Yeah that's fair, might as well. Updating 🏃

**Review by anncatton — APPROVED** [review]

**Review by xusandrew — APPROVED** [review]

**Pull request merged**